Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisationSuccess.cshtml

```diff
@@ -14,14 +19,12 @@
 }
 
     <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
-            <div class="govuk-panel govuk-panel--confirmation">
-                <h1 class="govuk-panel__title">
-                    @ViewData["Title"]
-                </h1>
-                <div class="govuk-panel__body">
-                    @($"{@StaticTextResource.OrganisationRegistration_JoinOrganisationSuccess_Message} {@organisationDetails.Name}")
-                </div>
-            </div>
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">
+                @ViewData["Title"]
+            </h1>
+
+        <p class="govuk-body">@StaticTextResource.OrganisationRegistration_JoinOrganisationSuccess_Message</p>
+        <p class="govuk-body">@Html.Raw(@StaticTextResource.Global_GoBackToMyAccount)</p>
```

Comment:
Since the `<a>` tag fully wraps the text here, I think it would be better to keep the link in the html, and only have the text of the link in the localized tag

Reply:
Good idea, done.